Description: Removing bundler dependecy
 Removing the bundler dependecy from tests.
 .
 ruby-redis-rack (1.5.0-1) UNRELEASED; urgency=medium
 .
   * Initial release (Closes: #nnnn)
Author: Balasankar C <balasankarc@autistici.org>


--- ruby-redis-rack-1.5.0.orig/test/test_helper.rb
+++ ruby-redis-rack-1.5.0/test/test_helper.rb
@@ -1,4 +1,4 @@
-require 'bundler/setup'
+#require 'bundler/setup'
 require 'minitest/autorun'
 require 'mocha/setup'
 require 'rack'
